lib/appinfo/application.php

```diff
@@ -91,30 +162,4 @@ public function __construct (array $urlParams = array()) {
 			return $c->query('ServerContainer')->getMemCacheFactory();
 		});
 	}
-
-	public function publicLinksAllowedToUseCollabora() {
```

Comment:
Please stop moving around code - this breaks review/rebase. THX

Reply:
done, sorry